tests/cluster/tests/23-multiple-slot-operations.tcl

```diff
@@ -66,3 +66,136 @@ test "Continuous slots distribution" {
     assert_match "* 13105-13500 14001-15000 16001-16383*" [$master5 CLUSTER NODES]
     assert_match "*13105 13500*14001 15000*16001 16383*" [$master5 CLUSTER SLOTS]
 }
+
+test "Add multiple slots with incorrect argument number" {
```

Comment:
This seems like a lot of tests for a small set of functionality. I would suggest we collapse this all down into a couple of tests that is testing boundary conditions against one node.

We could also simplify the tests by just asserting that the configuration on that one node hasn't changed.

Reply:
Follow your comment, I combine all boundary conditions into one test suite for every command, and assert the configuration on only one node.  Please take a look, Thanks